proto/frequenz/api/weather/weather.proto

```diff
@@ -90,9 +92,11 @@ message GetHistoricalWeatherForecastRequest {
   // period.
   google.protobuf.Timestamp end_ts = 4;
 
-  // Specifies the maximum number of forecasts to be returned in a single
-  // response.
-  int32 page_size = 5;
+  // The parameters define the 'page_size' and the 'page_token'. An inital
+  // query of a request will omit the 'page_token'. Subsequent queries of the
+  // same request must include the 'page_token' from the previous response.
+  optional frequenz.api.common.v1.pagination.PaginationParams
```

Comment:
We do not necessarily need to specify it as `optional`, because `PaginationParams` is optional by default (since it is not a scalar type).